source/api/commands/then.md

```diff
@@ -46,6 +46,8 @@ Pass a function that takes the previously yielded subject as its first argument.
 
 `.then()` is modeled identically to the way Promises work in JavaScript.  Whatever is returned from the callback function becomes the new subject and will flow into the next command (with the exception of `undefined`).
 
+Additionally, the last Cypress command in the callback function will be yielded as the new subject and flow into the next command if there is no `return`.
```

Comment:
`the command will be yielded` or the `result of the last Cypress command in the callback function will be yielded`?

Reply:
yeah, I knew I needed some review on this language. Was hard to write. Thanks!